toolchain/semantics/semantics_ir.cpp

```diff
@@ -286,6 +288,12 @@ auto SemanticsIR::StringifyType(SemanticsTypeId type_id,
 
     auto node = GetNode(step.node_id);
     switch (node.kind()) {
+      case SemanticsNodeKind::ArrayType: {
+        auto [bound_id, type_id] = node.GetAsArrayType();
+        out << "array [" << StringifyType(type_id) << "; "
```

Comment:
This function is trying to print the type as Carbon syntax, so we shouldn't have the `array ` prefix here.